cmd/commands/reset.go

```diff
@@ -0,0 +1,47 @@
+package commands
+
+import (
+	"os"
+	"path"
+
+	"github.com/urfave/cli"
+
+	tmcfg "github.com/tendermint/tendermint/config/tendermint"
+	types "github.com/tendermint/tendermint/types"
+)
+
+var UnsafeResetAllCmd = cli.Command{
+	Name:      "unsafe_reset_all",
+	Usage:     "Reset all blockchain data",
+	ArgsUsage: "",
+	Action: func(c *cli.Context) error {
+		return cmdUnsafeResetAll(c)
+	},
+}
+
+func cmdUnsafeResetAll(c *cli.Context) error {
```

Comment:
once all the cobra/cli stuff is merged we can go back through these and replace the ones that are just tendermint